frontend/src/app/service/type-converter/flowchart-converter.service.ts

```diff
@@ -74,43 +84,47 @@ export class FlowchartConverterService {
   }
 
   private convertWorkflowConstants(
-    workflow: WorkflowBaseItem,
+    workflow: WorkflowTransformation,
     flowchart: FlowchartConfiguration
   ): void {
-    const convertIoItem = (io: IOItem, isInput: boolean) => {
-      if (io.constant === undefined || io.constant === false) {
+    const convertConstant = (constant: Constant, isInput: boolean) => {
+      if (constant === undefined) {
         return;
       }
-      if (io.operator === undefined || io.connector === undefined) {
+      if (
+        constant.operator_id === undefined ||
+        constant.connector_id === undefined
+      ) {
         return;
       }
       const component = flowchart.components.find(
-        flowchartComponent => flowchartComponent.uuid === io.operator
+        flowchartComponent => flowchartComponent.uuid === constant.operator_id
       );
       if (component === undefined) {
         return;
       }
       const connector = (isInput ? component.inputs : component.outputs).find(
-        input => input.uuid === `${io.operator}_${io.connector}`
+        input =>
+          input.uuid === `${constant.operator_id}_${constant.connector_id}`
       );
       if (connector === undefined) {
         return;
       }
       connector.constant = true;
-      connector.value = io.constantValue.value;
+      connector.value = constant.value;
 
       flowchart.links = flowchart.links.filter(
```

Comment:
// TODO: Maybe links to constants can be removed from the API completely, since they are deleted here anyway.